test/test_helper.rb

```diff
@@ -68,6 +68,17 @@ def create_items_and_order
     Order.create(current_status: 'completed')
   end
 
+  def create_items_associated_with_orders
+    create_categories_and_items
+    order   = Order.create(current_status: 'completed')
+    order_2 = Order.create(current_status: 'completed')
```

Comment:
Useless assignment to variable - `order_2`. Did you mean `order`?<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.